apps/admin-gui/src/app/shared/components/user-detail-page/user-resources/user-resources.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  FacilitiesManagerService,
+  Facility, Resource,
+  ResourcesManagerService, RichResource, UsersManagerService, Vo,
+  VosManagerService
+} from '@perun-web-apps/perun/openapi';
+import { SelectionModel } from '@angular/cdk/collections';
+import { TABLE_ADMIN_USER_RESOURCES_LIST, TableConfigService } from '@perun-web-apps/config/table-config';
+import { ActivatedRoute } from '@angular/router';
+import { PageEvent } from '@angular/material/paginator';
+
+@Component({
+  selector: 'app-perun-web-apps-user-resources',
+  templateUrl: './user-resources.component.html',
+  styleUrls: ['./user-resources.component.scss']
+})
+export class UserResourcesComponent implements OnInit {
+
+  constructor(private userManager: UsersManagerService,
+              private tableConfigService: TableConfigService,
+              private route: ActivatedRoute) {
+  }
+
+  resources: RichResource[] = [];
+  loading: boolean;
+  pageSize: number;
+  tableId = TABLE_ADMIN_USER_RESOURCES_LIST;
+
+  ngOnInit(): void {
+    this.pageSize = this.tableConfigService.getTablePageSize(this.tableId);
+    this.refreshTable();
+  }
+
+  private refreshTable() {
+    this.loading = true;
+    this.route.parent.params.subscribe(parentParams => {
+      const userId = parentParams['userId'];
+      this.userManager.getAssignedRichResources(userId).subscribe(richResources => {
+        this.resources = richResources;
```

Comment:
I suggest you - before you make some PR, to try in GUI if everything is working, because you miss here:
`this.loading = false;`

Reply:
Please insert `      this.loading = false;` in the subscribe part of the code. It is because of that we want to show the page to the user after the asynchronous operation is done.